lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +159,22 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: replace-name-suffix
+    path: /name/{index}/suffix
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.suffix
+    description: Replace the `HumanName.suffix` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "op": "replace",
+        "path": "/name/0/suffix",
+        "value": ["Jr."]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `suffix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no suffix value is provided, the suffix on the name will be unset.</li>
+      </ul>
```

Comment:
let's add a new line at the end.

Reply:
added here https://github.com/cerner/fhir.cerner.com/commit/194a06ef023f0b84455cd6f6c0c46f35b338ee4c